src/audit/github_env.rs

```diff
@@ -72,7 +111,7 @@ mod tests {
     use crate::audit::github_env::GitHubEnv;
 
     #[test]
-    fn test_shell_patterns() {
+    fn test_exploitable_bash_patterns() {
```

Comment:
I think we can fold these two test functions into one, with `(case, expected: bool)` as the parametrization 🙂